support/corporates.md

```diff
@@ -26,22 +26,22 @@ If you are interested in becoming a Corporate Member or supporting one or more o
 
 ## Tibus Connect 
 
-<div style="float: right; padding: 0 20px 0 10px "><a href="http://www.tibus.com"><img src="{{site.base}}/assets/img/corp_logos/tibus.png"/></a></div>
+<div style="float: right; padding: 0 20px 0 10px "><a href="https://www.tibus.com"><img src="{{site.base}}/assets/img/corp_logos/tibus.png"/></a></div>
 
-[Tibus(Connect)](http://www.tibusconnect.com/) offer a full range of
+[Tibus(Connect)](https://www.tibusconnect.com/) offer a full range of
 connectivity solutions for business across the UK & Ireland.They are market
-partners of [Project Kelvin](http://www.tibusconnect.com/connect/project-
+partners of [Project Kelvin](https://www.tibusconnect.com/connect/project-
```

Comment:
https://www.tibus.com/

Reply:
This link is actually completely dead. I have removed it in a separate commit.